server/config/options.go

```diff
@@ -159,6 +160,12 @@ var DefaultConfig = Config{
 			ResponseTime: true,
 		},
 	},
+	Quota: QuotaConfig{
+		Enabled:         true,
+		RateLimit:       1000,        // requests per second
```

Comment:
in prod, 1000 is very high per account per worker, it should be less than 100 for a single account.

Reply:
Right, I think it's better to disable by default then, and put in dev and prod cluster config specific values.

Reply:
sg